2022-01-23 11:56:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 11:56:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 11:56:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 11:56:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 11:57:40 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 11:57:40 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 11:57:40 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 11:57:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 11:59:31 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 11:59:31 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 11:59:31 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 11:59:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:02:27 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:02:27 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:02:27 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:02:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:03:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:03:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:03:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:03:49 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:03:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:05:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:05:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:05:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:05:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:05:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:05:28 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:05:28 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:05:28 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:05:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:05:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:07:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:07:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:07:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:07:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:07:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:10:29 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:10:29 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:10:29 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:10:56 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:10:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:17:45 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:17:45 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:17:46 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:17:55 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:17:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:45:19 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:45:19 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:45:19 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:45:39 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:45:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:46:55 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:46:55 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:46:55 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:49:08 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:49:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:50:21 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:50:21 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:50:21 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:50:50 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:50:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:51:42 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:51:42 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:51:42 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:51:52 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:51:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 12:54:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:54:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 12:54:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:54:36 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 12:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:00:10 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:00:10 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:00:10 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:00:10 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:00:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:01:38 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:01:38 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:01:38 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:03:22 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:03:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "ACCOUNT" не найдена
Table "ACCOUNT" not found; SQL statement:
 DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:04:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:04:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:04:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:04:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:04:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:04:19 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:04:19 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:04:19 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:04:28 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:04:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:07:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:07:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:07:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:07:13 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:07:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:09:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:09:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:09:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:09:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:09:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:09:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:09:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
insert into hall(row, cell, session_id, available) values(?, ? ,? ,?) [42102-200]
2022-01-23 13:09:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:09:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "HALL" не найдена
Table "HALL" not found; SQL statement:
select * from hall ORDER BY id [42102-200]
2022-01-23 13:09:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "TICKET" не найдена
Table "TICKET" not found; SQL statement:
DELETE FROM ticket; DELETE FROM account; DELETE FROM hall [42102-200]
2022-01-23 13:16:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:16:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:16:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:16:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:16:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:16:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOG" не найдена
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOG [42102-200]
2022-01-23 13:16:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:16:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOG" не найдена
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT MAX(ORDEREXECUTED) FROM DATABASECHANGELOG [42102-200]
2022-01-23 13:17:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:17:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID SERIAL PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT AUTO_INCREMENT PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID SERIAL PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT AUTO_INCREMENT PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id SERIAL PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id int auto_increment primary key,
                         row int NOT NULL,
                         cell int NOT NULL,
                         session_id int not null,
                         available boolean
); [42001-200]
2022-01-23 13:21:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:21:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:22:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:22:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:22:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOG" не найдена
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOG [42102-200]
2022-01-23 13:22:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:22:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOG" не найдена
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT MAX(ORDEREXECUTED) FROM DATABASECHANGELOG [42102-200]
2022-01-23 13:22:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:22:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT AUTO_INCREMENT PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT AUTO_INCREMENT PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT AUTO_INCREMENT PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT AUTO_INCREMENT PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id int auto_increment primary key,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id int auto_increment primary key,
                         row int NOT NULL,
                         cell int NOT NULL,
                         session_id int not null,
                         available boolean
); [42001-200]
2022-01-23 13:31:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:31:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:31:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:31:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:31:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOG" не найдена
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOG [42102-200]
2022-01-23 13:31:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:31:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOG" не найдена
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT MAX(ORDEREXECUTED) FROM DATABASECHANGELOG [42102-200]
2022-01-23 13:31:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:31:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Неизвестный тип данных: "NOT"
Unknown data type: "NOT"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id IDENTITY NOT NULL PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id int auto_increment primary key,
                         row int NOT NULL,
                         cell int NOT NULL,
                         session_id int not null,
                         available boolean
); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5678)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:54)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:352)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:125)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1229)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1211)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:600)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:53)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:78)
	at liquibase.Liquibase.update(Liquibase.java:202)
	at liquibase.Liquibase.update(Liquibase.java:179)
	at liquibase.Liquibase.update(Liquibase.java:334)
	at org.liquibase.maven.plugins.LiquibaseUpdate.doUpdate(LiquibaseUpdate.java:33)
	at org.liquibase.maven.plugins.AbstractLiquibaseUpdateMojo.performLiquibaseTask(AbstractLiquibaseUpdateMojo.java:30)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:366)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
2022-01-23 13:33:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:33:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:33:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:33:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Неизвестный тип данных: "PRIMARY"
Unknown data type: "PRIMARY"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id int auto_increment primary key,
                         row int NOT NULL,
                         cell int NOT NULL,
                         session_id int not null,
                         available boolean
); [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5678)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:54)
	at org.h2.command.CommandList.update(CommandList.java:67)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:352)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:125)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1229)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1211)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:600)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:53)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:78)
	at liquibase.Liquibase.update(Liquibase.java:202)
	at liquibase.Liquibase.update(Liquibase.java:179)
	at liquibase.Liquibase.update(Liquibase.java:334)
	at org.liquibase.maven.plugins.LiquibaseUpdate.doUpdate(LiquibaseUpdate.java:33)
	at org.liquibase.maven.plugins.AbstractLiquibaseUpdateMojo.performLiquibaseTask(AbstractLiquibaseUpdateMojo.java:30)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:366)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
2022-01-23 13:36:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:36:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:36:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:36:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 13:42:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:42:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:43:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:43:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW2 INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW[*], CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW2 INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW[*], CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row2 INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 13:44:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:44:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:45:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW2 INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW[*], CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW3 INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW2 INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW[*], CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW3 INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row2 INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row3 INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 13:47:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:47:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:47:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 13:47:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 13:47:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


create TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

create TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 14:02:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROW[*] INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW, CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


CREATE TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        row INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

CREATE TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 14:11:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:11:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:11:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:11:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROWPLACE INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW[*], CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS TICKET (
                        ID INT IDENTITY PRIMARY KEY,
                        SESSION_ID INT NOT NULL,
                        ROWPLACE INT NOT NULL,
                        CELL INT NOT NULL,
                        ACCOUNT_ID INT NOT NULL REFERENCES ACCOUNT(ID),
                        CONSTRAINT CONSTRAINT_TICKET UNIQUE (SESSION_ID, ROW[*], CELL)
);

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


CREATE TABLE IF NOT EXISTS ticket (
                        id INT IDENTITY PRIMARY KEY,
                        session_id INT NOT NULL,
                        rowplace INT NOT NULL,
                        cell INT NOT NULL,
                        account_id INT NOT NULL REFERENCES account(id),
                        CONSTRAINT constraint_ticket UNIQUE (session_id, row, cell)
);

CREATE TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 14:12:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Синтаксическая ошибка в выражении SQL "

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW[*] INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; ожидалось "identifier"
Syntax error in SQL statement "

CREATE TABLE IF NOT EXISTS HALL (
                         ID INT IDENTITY PRIMARY KEY,
                         ROW[*] INT NOT NULL,
                         CELL INT NOT NULL,
                         SESSION_ID INT NOT NULL,
                         AVAILABLE BOOLEAN
);"; expected "identifier"; SQL statement:


CREATE TABLE IF NOT EXISTS hall (
                         id INT IDENTITY PRIMARY KEY,
                         row INT NOT NULL,
                         cell INT NOT NULL,
                         session_id INT not null,
                         available BOOLEAN
); [42001-200]
2022-01-23 14:14:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-01-23 14:14:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-01-23 14:14:11 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-01-23 14:14:45 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:246)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:149)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:97)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:67)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:316)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 50 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 44 more
2022-01-23 14:14:49 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-01-23 14:15:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:15:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:17:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:17:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2022-01-23 14:17:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2022-01-23 14:17:54 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-01-23 14:19:55 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:246)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:149)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:97)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:67)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:316)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 50 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 44 more
2022-01-23 14:20:13 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:246)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:149)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:97)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:67)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:316)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 50 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 44 more
2022-01-23 14:20:23 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:246)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:149)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:97)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:67)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:316)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 50 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 44 more
2022-01-23 14:20:25 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Столбец "row" не найден
Column "row" not found [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:352)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at org.apache.commons.dbcp2.DelegatingResultSet.getInt(DelegatingResultSet.java:633)
	at ru.job4j.store.DbStore.hall(DbStore.java:119)
	at ru.job4j.store.DbStoreTest.whenAllTicket(DbStoreTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-01-23 14:20:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:20:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:21:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:33:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:33:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 14:34:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:00:48 database: flush
org.h2.message.DbException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at org.h2.engine.Database.getPageStore(Database.java:2659)
	at org.h2.engine.Database.open(Database.java:675)
	at org.h2.engine.Database.openDatabase(Database.java:307)
	at org.h2.engine.Database.<init>(Database.java:301)
	at org.h2.engine.Engine.openSession(Engine.java:74)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at liquibase.database.DatabaseFactory.openConnection(DatabaseFactory.java:246)
	at liquibase.database.DatabaseFactory.openDatabase(DatabaseFactory.java:149)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:97)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:67)
	at org.liquibase.maven.plugins.AbstractLiquibaseMojo.execute(AbstractLiquibaseMojo.java:316)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]" [50000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 50 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/projects/job4j_cinema/testdb.mv.db [1.4.200/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 44 more
2022-01-23 15:01:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:01:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:01:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:11:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:11:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:11:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:43:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:43:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:44:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:52:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 15:52:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 16:13:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 16:13:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-23 16:13:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:12:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:12:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:12:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:14:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:14:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:14:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:17:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:17:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
2022-01-24 18:17:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Таблица "DATABASECHANGELOGLOCK" не найдена
Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM DATABASECHANGELOGLOCK [42102-200]
